src/main/java/io/confluent/castle/role/BrokerRole.java

```diff
@@ -34,7 +34,8 @@
 import java.util.Map;
 
 public class BrokerRole implements Role {
-    public static final String KAFKA_CLASS_NAME = "kafka.Kafka";
+    // Handle both kafka.Kafka and SupportedKafka
+    public static final String KAFKA_CLASS_NAME = "Kafka";
```

Comment:
We might have to eventually think of a better solution here, but this is OK for now.